services/blockchain-indexer/shared/dataService/business/events.js

```diff
@@ -73,6 +86,14 @@ const getEvents = async (params) => {
 		params = normalizeRangeParam(params, 'timestamp');
 	}
 
+	if (params.order) {
+		const { order, ...remParams } = params;
+		params = remParams;
+		params.sort = params.sort
+			? params.sort.concat(',', order)
+			: order;
+	}
```

Comment:
Instead of the using two parameters for sorting, could we accept multiple sort columns and if none of these contains `index` append `index:asc` to the sort param?